Packs/ZeroFox/ReleaseNotes/1_3_3.md

```diff
@@ -0,0 +1,6 @@
+#### Integrations
+
+##### ZeroFox
+
+- Removed the **Fetch Limit** parameter.
+- Improved implementation of ***fetch_incidents*** to avoid duplication or miss in fetching the incidents.
```

Comment:
```suggestion
- Improved implementation of ***fetch_incidents*** to avoid duplication or miss in fetching the incidents.
- Updated the Docker image to: *demisto/python3:3.10.14.98471*.
```

Reply:
add the docker image update here

Reply:
@DNRRomero Missed that one :)